{
  "id": "CVE-2023-28609",
  "assigner": "cve@mitre.org",
  "Published": "2023-03-18T20:15:11.667Z",
  "Modified": "2023-03-20T01:46:58.537Z",
  "last-modified": "2023-03-20T01:46:58.537Z",
  "summary": "api/auth.go in Ansible Semaphore before 2.8.89 mishandles authentication.",
  "access": {
    "authentication": null,
    "complexity": null,
    "vector": null
  },
  "impact": {
    "availability": null,
    "confidentiality": null,
    "integrity": null
  },
  "cvss": null,
  "exploitabilityScore": null,
  "impactScore": null,
  "cvss-vector": null,
  "references": [
    "https://github.com/ansible-semaphore/semaphore/commit/3e4a62b7f2b1ef0660c9fb839818a53c80a5a8b1",
    "https://github.com/ansible-semaphore/semaphore/releases/tag/v2.8.89"
  ],
  "vulnerable_configuration": [],
  "vulnerable_product": [],
  "vendors": [],
  "products": [],
  "vulnerable_product_stems": [],
  "vulnerable_configuration_stems": [],
  "cwe": "Unknown",
  "vulnerable_configuration_cpe_2_2": []
}